status-im/status-react
# feat: set spending cap screen

fixes #20336  

### Summary

This PR implements Set Spending Cap screen, no backend integration yet and some data is hardcoded for now, but UI should be functional and most of the logic should be in place

<img src="https://github.com/user-attachments/assets/41d84263-05f0-463f-971a-20a655571e74" width="300" />
<img src="https://github.com/user-attachments/assets/d4817920-0f1a-4805-ae7b-f4300597fbb8" width="300" />

#### Platforms

- Android
- iOS

#### Areas that maybe impacted

##### Functional

- wallet / transactions

### Steps to test

- Open Status
- Log in
- Open wallet
- Select an account
- Tap on Swap
- Select an asset in asset to pay screen
- Tap on Set spending cap button to navigate to Set spending cap screen
- Verify UI looks as expected

status: ready
